CHERICC_Fat.bsv

```diff
@@ -1514,7 +1514,7 @@ instance CHERICap #(CapReg, 0, 0, CapAddrW, CapW, TSub#(MW, 2));
 
   // Assert that the encoding is valid
   //////////////////////////////////////////////////////////////////////////////
-  function isDerivable (cap) =
+  function hasLegalBounds (cap) =
```

Comment:
I'd be tempted to split out the reserved bit zero checks into `hasLegalReservedBits`, and then or that into the `isLegal`?